data_handling/munge_ansirh.py

```diff
@@ -0,0 +1,124 @@
+"""
+Munge ANSIRH data into JSON files binned by state.
+
+Author(s): Kate Habich
+"""
+
+import pandas as pd
+import json
+from .clean_ansirh import clean_ansirh
+
+
+def main_ansirh():
```

Comment:
For consistency with the other files/convention, it may be nice to just call this function "main" (since it's in the munge_ansirh.py file, it's clear that it's the main for ansirh).